docs/api.md

```diff
@@ -446,3 +446,37 @@ Public API supported by the BCMP configuration module is as follows:
 
   :returns: true if delete message is sent properly, false otherwise
 ```
+
+## Time
+Bristlemouth offers a time API that can be used to obtain or set the current time on a device.
+
+Time messages are sent over the Bristlemouth Control Messaging Protocol (BCMP).
+
+The following header file path must be included when calling BCMP time API:
+
+```
+#include "messages/time.h"
+```
+
+Public API supported by the BCMP time module is as follows:
+
+```{eval-rst}
+.. cpp:function:: BmErr bcmp_time_get_time(uint64_t target_node_id)
+
+  Get the current time from a target node.
+
+  :param target_node_id: Target node id to obtain the current time from
+
+  :returns: BmErr value
```

Comment:
Maybe it is worthwhile to put certain conditions/return values to help users navigate errors?

Reply:
Good idea! Added!